www/assets/css/wpt-header.css

```diff
@@ -416,7 +416,7 @@ wpt-header .wptheader_nav_menu details[open] summary span:after {
 
 @media (min-width: 86em) {
   wpt-header {
-    z-index: 999999;
+    z-index: 1000;
```

Comment:
This is going to change header z-index everywhere, are you sure it does not break anything?